modules/jaeger-config-collector.adoc

```diff
@@ -36,4 +40,46 @@ The collectors are stateless and thus many instances of Jaeger Collector can be
 |log-level:
 |Logging level for the collector.
 |Possible values: `trace`, `debug`, `info`, `warning`, `error`, `fatal`, `panic`.
+
+|maxReplicas:
+|Specifies the maximum number of replicas to create when autoscaling the Collector.
+|Integer, for example, `100`.
+
+|num-workers:
+|The number of workers pulling from the queue.
+|Integer, for example, `50`.
+
+|queue-size:
+|The size of the Collector queue.
+|Integer, for example, `2000`.
+
+|replicas:
+|Specifies the number of Collector replicas to create.
+|Integer, for example, `5`.
 |===
+
+
+== Configuring the Collector for autoscaling
+
```

Comment:
I think both the Collector and Ingester sections could benefit from a sentence that tells me What does configuring the Collector and Ingester to autoscale on demand do for me? Why would I want to do that?